content/en/tools/cdl/cdl-overview.md

```diff
@@ -96,7 +96,7 @@ The Ledger Evolution view also has an additional Privacy overlay represented as
 
 With this view, you have insight on privacy from the design phase. An unintended privacy leak can be a show stopper for a CorDapp. It's best to consider this early on, not at the point where your customer's information security team won't sign off the deployment.
```

Comment:
Not one of your changes but I think "show stopper" is used in the wrong context here. I think it's all one word too, _showstopper._

[ a stage act, etc, that receives so much applause as to interrupt the performance.]

Reply:
It can mean an obstacle to further progress, which is probably more applicable here. However happy to change to showstopper.